src/scripts/resource-channels.js

```diff
@@ -1,57 +1,70 @@
 const { mute } = require("./users/permissions");
 const { logMessageDate } = require("./utilities");
+const { User } = require("./users/user-utilities");
 
 const timeLimit = 120000;
 const warnOn = 3;
 const muteOn = 4;
+global.temp = {};
```

Comment:
After looking at it again, it doesn't! The timeouts just need to be attached to an object in the global scope of the module. They just needed to be cleared on separate messages.

I was looking for a way to keep the User objects structure clean and easy to debug (since `setTimeout()` actually returns an entire timeout object in node).

I've created a separate object for it with a3bdd6c. Is this a cleaner way of structuring it, or do you think I'm missing something and there's a better way?

Reply:
I'm not sure of a better way to do it, I think it looks great as it is! 👍